### PR TITLE
Feature/search feature admunits

### PR DESCRIPTION
**Pull request by gcapurso**

### CODE REVIEW

**Review by Windvis — CHANGES_REQUESTED** [review]
Good job! I only added some tiny / nitpicky remarks!

**Comment by Windvis** [comment]
There also seem to be some linting issues, so be sure to run `npm run lint:fix`.

File: app/templates/people/index.hbs [hunk 1]
```diff
@@ -7,15 +7,15 @@
                   class="au-c-form"
                 >
                     <p>
-                      <AuLabel for="filter-given-name">Given name</AuLabel>
+                      <AuLabel for="filter-given-name">Voornaam</AuLabel>
```
[the remark this answers is not in the document]
Comment (gcapurso) [reply app/templates/people/index.hbs:10]: Thank you, You are right.

File: app/routes/administrative-units/index.js [hunk 2]
```diff
@@ -1,10 +1,36 @@
...
+      query['filter[primary-site][address][municipality]'] =
+        params.municipality;
+    }
+
+    if (params.organizationStatus) {
+      query['filter[organization-status][:id:]'] = params.organizationStatus;
+    }
+
+    return hash({
+      classifications: this.store.findAll(
```
[the remark this answers is not in the document]
Comment (gcapurso) [reply app/routes/administrative-units/index.js:28]: The updated data are available here: https://github.com/lblod/organisaties-data

**Review by Windvis — APPROVED** [review]

**Pull request merged**